newsroom/search_config.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Dict, Any, KeysView, Type
+import logging
+from copy import deepcopy
+
+from superdesk.resource import not_analyzed
+from newsroom.factory.app import BaseNewsroomApp
+from newsroom import Resource
+
+
+logger = logging.getLogger(__name__)
+nested_agg_groups = {}
+
+
+def is_search_field_nested(resource_type: str, field: str):
+    """Returns ``True`` if the ``resource_type`` is configured for nested search group"""
+
+    return field in (nested_agg_groups.get(resource_type) or {})
+
+
+def init_nested_aggregation(
```

Comment:
just wondering why doing it dynamically, imo the groups config is something we might set/change later but would be good to have the mapping on an instance from the start so we don't have to reindex when changing the config